src/platform/forms/save-in-progress/api.js

```diff
@@ -56,57 +36,17 @@ export function saveFormApi(
     },
     formData,
   });
-  const csrfTokenStored = localStorage.getItem('csrfToken');
   const apiUrl = inProgressApi(formId);
   const saveFormApiHeaders = {
-    'Content-Type': 'application/json',
     'X-Key-Inflection': 'camel',
-    'Source-App-Name': window.appName,
-    'X-CSRF-Token': csrfTokenStored,
```

Comment:
Can you explain what removing these headers has to do with switching from `fetch()` to `apiRequest()`?

Reply:
Sure. The `apiRequest` already uses these headers, aka less duplication of code/DRY